include/detect_breakends/junction_detection.hpp

```diff
@@ -35,7 +35,7 @@
  */
 void detect_junctions_in_alignment_file(const std::filesystem::path & alignment_file_path,
                                         const std::filesystem::path & insertion_file_path,
-                                        const std::vector<uint8_t> methods,
+                                        const std::vector<detecting_methods> methods,
```

Comment:
The name `detecting_methods` does not sound right. Maybe rename it to `detection_methods` if it's not too much of a hassle?